spec/features/search/index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'Search Index Page' do
```

Comment:
This could be tested a little more thoroughly - checking what is in the page for example after clicking the "Find Nearest Station" button.